addon/components/ember-tableish-headers.js

```diff
@@ -1,91 +1,75 @@
-import { A as emberArray } from '@ember/array';
-import { computed } from '@ember/object';
-import Component from '@ember/component';
-import layout from '../templates/components/ember-tableish-headers';
+import Component from '@glimmer/component';
 import { getOwner } from '@ember/application';
+import { action } from '@ember/object';
+import { tracked } from 'tracked-built-ins';
 
-export default Component.extend({
-  layout,
-  classNames: ['ember-tableish-headers'],
+export default class EmberTableishHeaders extends Component {
+  cols = tracked(Set);
 
-  nonce: computed({
-    get() {
-      const config = getOwner(this).resolveRegistration('config:environment');
+  get nonce() {
+    const config = getOwner(this).resolveRegistration('config:environment');
 
-      const configNonce = config['ember-tableish-csp-nonce'];
-      if (configNonce) {
-        return configNonce;
-      }
-
-      const metaNonce = document.querySelector(
-        'meta[name="ember-tableish-csp-nonce"]'
-      );
-
-      if (metaNonce) {
-        return metaNonce.content;
-      }
+    const configNonce = config['ember-tableish-csp-nonce'];
+    if (configNonce) {
+      return configNonce;
+    }
 
-      return null;
-    },
+    const metaNonce = document.querySelector(
+      'meta[name="ember-tableish-csp-nonce"]'
+    );
 
-    set(k, v) {
-      return v;
+    if (metaNonce) {
+      return metaNonce.content;
     }
-  }),
 
-  gridColumns: computed('cols.@each.width', {
-    get() {
-      // see https://css-tricks.com/preventing-a-grid-blowout for why we need to use minmax
-      return this.cols.map(c => `minmax(0, ${c.width})`).join(' ');
-    }
-  }),
+    return null;
+  }
 
-  msGridColumns: computed('gridColumns', 'table.columnGap', {
-    get() {
-      const { columnGap } = this.table;
+  get gridColumns() {
+    // see https://css-tricks.com/preventing-a-grid-blowout for why we need to use minmax
+    const strs = [];
+    this.cols.forEach((c) => strs.push(`minmax(0, ${c.width})`));
```

Comment:
I'd probably stick with the `.map` impl or use `for of` if we don't want the closure for... reasons